docs/multicast-guide.md

```diff
@@ -0,0 +1,153 @@
+Example output of the command:
+
+```bash
+kubectl get multicastgroups
+GROUP       PODS
+225.1.2.3   default/mcjoin, namespace/pod
+224.5.6.4   default/mcjoin
+```
+
+### Inbound and outbound multicast traffic statistics
+
+`antctl` supports printing multicast traffic statistics of Pods. Please refer to the corresponding [antctl user guide section](antctl.md#multicast-commands).
+
+### Multicast NetworkPolicy statistics
+
+The [Antrea NetworkPolicyStats feature](feature-gates.md#networkpolicystats) also supports multicast NetworkPolices.
+
+## Use case example
+
+This section will take multicast video streaming as an example to demonstrate how multicast works
+with Antrea. In this example, [VLC](https://www.videolan.org/vlc/) multimedia tools are used to
+generate and consume multicast video streams.
+
+To start a video streaming server, we start a VLC Pod to stream a sample video
+to the multicast IP address `239.255.12.42` with TTL 6.
+
+```bash
+kubectl run -i --tty --image=quay.io/galexrt/vlc:latest vlc-sender -- --intf ncurses --vout dummy --aout dummy 'https://upload.wikimedia.org/wikipedia/commons/transcoded/2/26/Bees_on_flowers.webm/Bees_on_flowers.webm.120p.vp9.webm' --sout udp:239.255.12.42 --ttl 6 --repeat
+```
+
+You can verify multicast traffic is sent out from this Pod by running
+`antctl get podmulticaststats` in the `antrea-agent` Pod on the local Node,
+which indicates the VLC Pod is sending out multicast video streams.
+
+You can also check the multicast routes on the Node by running command `ip mroute`,
+which should print the following route for forwarding the multicast traffic from
+the Antrea gateway interface to the transport interface.
+
+```bash
+ip mroute
+(pod-ip, 239.255.12.42)     Iif: antrea-gw0 Oifs: transport interfaces State: resolved
+```
+
+We also create a VLC Pod to be the receiver with the following command:
+
+```bash
+kubectl run -i --tty --image=quay.io/galexrt/vlc:latest vlc-receiver -- --intf ncurses --vout dummy --aout dummy udp://@239.255.12.42 --repeat
+```
+
+It's expected to see inbound multicast traffic to this Pod by running
+`antctl get podmulticaststats` in the local `antrea-agent` Pod,
+which indicates the VLC Pod is receiving the video stream.
+
+Also, the `kubectl get multicastgroups` command will show that `vlc-receiver`
+has joined multicast group `239.255.12.42`.
+
+## Limitations
+
+This feature is currently supported only for IPv4 Linux clusters.
+Support for Windows and IPv6 will be added in the future.
+
+### Encap mode
+
+Configuration option `multicastInterfaces` is not supported with encap mode.
+Multicast packets in encap mode are SNATed and forwarded to the transport interface only.
+
+### Traffic in local network control block
+
+Multicast IPs in [Local Network Control Block](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xhtml#multicast-addresses-1) (224.0.0.0/24)
+should work only in Antrea encap mode. Multicast traffic destined for those addresses
```

Comment:
should work only in Antrea encap mode? I feel it's "can only work in encap mode". @jianjuns

Reply:
I don't understand your point here. Could you explain more?

Basically we has tested some simple cases for MDNS(224.0.0.251) and it works as expected in encap mode. The other multicast service running in 224.0.0.0/24 are not tested comprehensively, therefore may not work as expected.

Reply:
@tnqn means changing "should" to "can". I agree "can" sounds better here.

Reply:
updated. Thanks